rero_ils/modules/loans/api_views.py

```diff
@@ -34,13 +35,23 @@
 )
 
 
+@api_blueprint.route('/<loan_pid>/circulation_policy', methods=['GET'])
+@check_logged_as_librarian
+def dump_loan_circulation_policies(loan_pid):
```

Comment:
I imagine we can only have one policy for each loan: `dump_loan_circulation_policies` -> `dump_loan_circulation_policy`